<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutorizeWindow</class>
 <widget class="QMainWindow" name="AutorizeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="welcome_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>48</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Добро пожаловать!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>1200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Для авторизации введите имя</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="OK_Button">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>560</y>
      <width>350</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ОК</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RESET_Button">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>560</y>
      <width>350</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>СБРОС</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="Keyboad_Box">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>370</y>
      <width>600</width>
      <height>400</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>160</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>160</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>280</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>280</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_canel">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>40</y>
       <width>150</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clear">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>160</y>
       <width>150</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Корректировка</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_enter">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>280</y>
       <width>150</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ввод</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="textInputBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>300</y>
      <width>550</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
